examples/TurnOnTechnibelAC/TurnOnTechnibelAC.ino

```diff
@@ -0,0 +1,69 @@
+/* Copyright 2017, 2018 David Conran
```

Comment:
Please remove the example entirely. We already have numerous `TurnOnBlahAC.ino` examples, of which this is pretty similar. Adding it means our build times for the libraries gets increased for not much real value.